README.md

```diff
@@ -38,31 +38,28 @@ Oxipng follows Semantic Versioning.
 
 ## Usage
 
-Oxipng is a command-line utility. Basic usage looks similar to the following:
+Oxipng is a command-line utility. An example suitable for web usage may be the following:
 
 ```
-oxipng -o 4 -i 1 --strip safe *.png
+oxipng -o 4 --alpha --strip safe *.png
 ```
 
 The most commonly used options are as follows:
 
-- Optimization: `-o 1` through `-o 6`, lower is faster, higher is better compression.
-  The default (`-o 2`) is sufficiently fast on a modern CPU and provides 30-50% compression
-  gains over an unoptimized PNG. `-o 4` is 6 times slower than `-o 2` but can provide 5-10%
-  extra compression over `-o 2`. Using any setting higher than `-o 4` is unlikely
-  to give any extra compression gains and is not recommended.
-- Interlacing: `-i 1` will enable [Adam7](https://en.wikipedia.org/wiki/Adam7_algorithm)
-  PNG interlacing on any images that are processed. `-i 0` will remove interlacing from all
-  processed images. Not specifying either will keep the same interlacing state as the
-  input image. Note: Interlacing can add 25-50% to the size of an optimized image. Only use
-  it if you believe the benefits outweigh the costs for your use case.
+- Optimization: `-o 0` through `-o 6` (or `-o max`), lower is faster, higher is better compression.
+  The default (`-o 2`) is quite fast and provides good compression. Higher levels can be notably
+  better but generally have increasingly diminishing returns.
+- Alpha: `--alpha` can improve compression of images with transparency, by altering the color
+  values of fully transparent pixels. This is generally recommended, but take care as this is
+  technically a lossy transformation and may be unsuitable for some specific applications.
 - Strip: Used to remove metadata info from processed images. Used via `--strip [safe,all]`.
   Can save a few kilobytes if you don't need the metadata. "Safe" removes only metadata that
   will never affect rendering of the image. "All" removes all metadata that is not critical
   to the image. You can also pass a comma-separated list of specific metadata chunks to remove.
   `-s` can be used as a shorthand for `--strip safe`.
 
-More advanced options can be found by running `oxipng -h`.
+More advanced options can be found by running `oxipng -h`, or viewed [here](MANUAL.txt).
```

Comment:
Not related to this line, but I think the deploy workflow needs to include this file too maybe?

https://github.com/shssoichiro/oxipng/blob/master/.github/workflows/deploy.yml#L57

Reply:
Good point, I'll mention that in #563.

Reply:
Please add a note:
- Comment on parameters being case sensitive.
- Example of short vs long parameter usage, e.g. `-Psavvo6` == `--pretend --strip=safe --alpha --verbose --verbose --opt=6`.
This fixes #539.

Also suggest pointing user to `--help` (not `-h`), for extended help.

Reply:
@ace-dent done